neuro_extras/main.py

```diff
@@ -440,6 +388,9 @@ async def _nonstorage_cp(source_url: URL, destination_url: URL) -> None:
             str(source_url),
             str(destination_url),
         ]
+        if remove_source:
```

Comment:
redundant?

Reply:
No, that's a bug I forgot to take care of, thanks!

We should delete files manually, as rclone doesn't have an option rsync has.